src/Content/Conversation.php

```diff
@@ -154,7 +154,7 @@ public function builtinActivityPuller(array $activity, array &$conv_responses)
 				}
 
 				// Skip when the causer of the parent is the same as the author of the announce
-				if (($verb == Activity::ANNOUNCE) && !empty($thread_parent['causer-id'] && ($thread_parent['causer-id'] == $activity['author-id']))) {
+				if ($verb == Activity::ANNOUNCE && !empty($thread_parent['causer-id']) && $thread_parent['causer-id'] == $activity['author-id']) {
```

Comment:
So that everybody is happy.
```suggestion
				if (($verb == Activity::ANNOUNCE) && !empty($thread_parent['causer-id']) && ($thread_parent['causer-id'] == $activity['author-id'])) {
```

Reply:
I have added them.